src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1382,6 +1382,8 @@ struct SurfacesData : BaseGlobalStruct
     std::vector<int> AllExtSolWinWithFrameSurfaceList; // List of all exterior solar window surfaces with a frame and divider
     std::vector<int> AllHTKivaSurfaceList;             // List of all Kiva foundation surfaces
     std::vector<int> AllSurfaceListReportOrder;        // List of all surfaces - output reporting order
+    std::vector<int> AllVaryThermalAbsOpaqSurfaceList; // List of all opaque exterior surfaces with dynamic coating
```

Comment:
Yes, this is what I meant. :+1:

Reply:
:)

Reply:
This could become a single list.

Reply:
right, since we combined the idd objects